### PR TITLE
Security section enhancements

### PR DESCRIPTION
**Pull request by Holmistr**

### CODE REVIEW

File: src/IntegratingOpenSSL.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Below sections explain these steps.
 
 ### Installing OpenSSL
 
-Install OpenSSL. Make sure that you are installing 1.0.1 or 1.0.2 releases. Please refer to its documentation at 
+Install OpenSSL. Make sure that you are installing 1.0.1 or newer release. Please refer to its documentation at
```
Comment (pveentjer) [inline src/IntegratingOpenSSL.md]: I don't know about newer. 1.0.1 and 1.0.2 work fine.

File: src/Security.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ You can enable security declaratively or programmatically, as shown below.
 
 ```xml
 <hazelcast xsi:schemaLocation="http://www.hazelcast.com/schema/config
-    http://www.hazelcast.com/schema/config/hazelcast-config-3.3.xsd"
+    https://hazelcast.com/schema/config/hazelcast-config-3.9.xsd"
```
Comment (pveentjer) [inline src/Security.md]: Why the switch to https?

Comment (Holmistr) [inline src/Security.md]: Http version will still redirect you to Https, I didn't see any benefit in not changing it.

**Comment by Holmistr** [comment]
@pveentjer In my email, I see that you commented "I don't know about newer. 1.0.1 and 1.0.2 work fine.", but somehow I don't see it here now. However, I changed it because there is also new 1.1 version, which is btw installed default on systems like RHEL7, and it works with it too as well. I wanted to avoid user's feeling that he HAS to downgrade.

**Comment by pveentjer** [comment]
The 1.1 is fine; I checked Netty.

File: src/Security.md (hunk 2 above)
Comment (pveentjer) [inline src/Security.md]: In the reference manual we don't make use of https; so from a consistency point of view it is best to keep it at http.

**Comment by Holmistr** [comment]
@pveentjer Ok, I addressed your comment and fixed it.

File: src/Security.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ You can enable security declaratively or programmatically, as shown below.
 
 ```xml
 <hazelcast xsi:schemaLocation="http://www.hazelcast.com/schema/config
-    http://www.hazelcast.com/schema/config/hazelcast-config-3.3.xsd"
```
Comment (pveentjer) [inline src/Security.md]: I don't see 'http://hazelcast.com' in the rest of the manual

**Comment by Holmistr** [comment]
@pveentjer Ok, fixed according to your latest comment. @Serdaro Does it look to you OK as well?

**Comment by Holmistr** [comment]
@pveentjer @Serdaro I updated the PR once again with a note that OpenSSL is currently not working on IBM JDK.

**Comment by Serdaro** [comment]
@pveentjer , Could you please confirm the IBM JDK statement? I think there was a discussion about this in the past.

**Comment by pveentjer** [comment]
IBM doesn't work with OpenSSL currently.

**Pull request merged**